README.md

```diff
@@ -5,26 +5,85 @@ on Microsoft Azure REST APIs.
+| [redis](/arm/redis)                                           | [Redis Cache](https://azure.microsoft.com/services/cache/)                                                            |
+| [features](/resources/features)                               | [Feature Exposure Control](https://msdn.microsoft.com/library/azure/mt592690.aspx)                                    |
+| [locks](/resources/locks)                                     | [Lock resources](https://azure.microsoft.com/documentation/articles/resource-group-lock-resources/)                   |
+| [policy](/arm/resources/policy)                               | [Resources Policies](https://azure.microsoft.com/documentation/articles/resource-manager-policy/)                     |
+| [resources](/arm/resources/resources)                         | [Resource Manager](https://azure.microsoft.com/documentation/articles/resource-group-overview/)                       |
+| [subscriptions](/arm/resources/subscriptions)                 | [Subscriptions](https://msdn.microsoft.com/library/azure/dn790566.aspx)                                               |
+| [scheduler](/arm/scheduler)                                   | [Scheduler](https://azure.microsoft.com/services/scheduler/)                                                          |
+| [search](/arm/search)                                         | [Search](https://azure.microsoft.com/services/search/)                                                                |
+| [servicebus](/arm/servicebus)                                 | [Service Bus](https://azure.microsoft.com/services/service-bus/)                                                      |
+| [sql](/arm/sql)                                               | [SQL Database](https://azure.microsoft.com/services/sql-database/)                                                    |
+| [storage](/arm/storage)                                       | [Storage Accounts and usage](https://azure.microsoft.com/documentation/articles/storage-create-storage-account/)      |
+| [trafficmanager](/arm/trafficmanager)                         | [Traffic Manager](https://azure.microsoft.com/services/traffic-manager/)                                              |
+| [web](/arm/web)                                               | [Web Apps](https://azure.microsoft.com/services/app-service/web/)                                                     |
+
+## Azure Service Management (ASM), aka classic deployment
+
+| Package                                                       | Service                                                                                                                                                                                                                                                               |
```

Comment:
The table approach is not working. It's not clear to me that `management/` is at the root and others are its subdirectories.

Also ask yourself, how do you expect  users to go read `arm/README.md` or `management/README.md`. It's very hard to discover.

![image](https://cloud.githubusercontent.com/assets/159209/18015639/660fbbd0-6b7d-11e6-8ed9-b401ecdd0de3.png)